CHANGELOG.md

```diff
@@ -3,6 +3,52 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.25](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.24...v4.0.0-alpha.25)
+
+_Apr 14, 2021_
+
+Big thanks to the 5 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add boolean column type @m4theushw
+- ⚡️ Update to React 17 (#1331) @m4theushw
+- 🕹 Make column header cells focusable @DanailH
```

Comment:
How about we bundle the accessibility fixes under one line? I believe we did two fixes. We fix the roving index and include the header in the keyboard navigation. ♿ Is the icons we have used in the past to describe a11y improvements.